erdkotlin/src/main/java/com/elrond/erdkotlin/domain/transaction/models/TransactionInfo.kt

```diff
@@ -0,0 +1,24 @@
+package com.elrond.erdkotlin.domain.transaction.models
+
+import com.elrond.erdkotlin.domain.wallet.models.Address
+import java.math.BigInteger
+
+data class TransactionInfo(
```

Comment:
Isn't `TransactionOnNetwork` sufficient?

Reply:
This one is tricky:

My implementation of `TransactionOnNetwork` and `TransactionInfo` are based on the documentation and on my personal tests of the API.
`TransactionOnNetwork` is for "Get Address Transactions"
`TransactionInfo` is for "Get Transaction"

I have noticed so many differences between those two endpoints that I decided to create two types.

**Fields returned by "Get Address Transactions" and not by "Get Transaction"**
> - hash
> - HyperblockHash
> - timestamp
> - gasUsed
> - receiverShard
> - senderShard
> - searchOrder
> - fee
> - scResults

**Fields returned by "Get Transaction"  and not by "Get Address Transactions"**
> - type
> - blockHash
> - SourceShard
> - DestinationShard
> - MiniblockType
> - HyperblockNonce
> - HyperblockHash
> - epoch
> - blockNonce

Moreover, after looking at the Golang implementation of `TransactionOnNetwork` I noticed even more differences 
It doesn't know about any of these fields (which I can find in `Get Address Transactions` and `Get Transaction`)
> - blockHash
> - epoch
> - blockNonce
> - timestamp
> - gasUsed
> - receiverShard
> - senderShard
> - searchOrder
> - fee
> - scResults
> - round

This is why, as a first step, I have chosen this implementation.
I am, obviously open to any kind of guidance regarding this subject 😅

Reply:
All right :heavy_check_mark: 

They are indeed different, since the „get transactions of address” is backed by a different provider (it relies on Elastic Search data under the hood). 

The endpoints will evolve to be more consistent (in the near future), your code can stay as it is for the moment.